packages/@o3r/design/src/core/design-token/renderers/design-token/design-token-updater.renderers.ts

```diff
@@ -0,0 +1,25 @@
+import type { DesignContentFileUpdater } from '../design-token.renderer.interface';
+
+/** Options for {@link getDesignTokenStyleContentUpdater} */
+export interface DesignTokenStyleContentUpdaterOptions {
+}
+
+const mergeValue = (tokenObject: Record<string, any>, newValue: Record<string, any>) => {
+  return Object.entries(newValue)
+    .reduce((acc, [key, value]) => {
+      acc[key] = acc[key] ? mergeValue(acc[key], value) : value;
+      return acc;
+    }, { ...tokenObject });
+};
+
+/**
+ * Retrieve a Content Updater function for Metadata generator
+ * @param _options
+ */
+export const getDesignTokenStyleContentUpdater = (_options?: DesignTokenStyleContentUpdaterOptions): DesignContentFileUpdater => {
+  return (variables: string[]) => {
+    const nodes: {[x: string]: any}[] = JSON.parse(`[${variables.join(',')}]`);
```

Comment:
Why not using `Record<string, any>[]` to be consistent with the line below or the opposite ?

Reply:
No reason, I used it in the beginning and forgot to fix this one